packages/odyssey-react-mui/src/Checkbox.tsx

```diff
@@ -160,9 +181,14 @@ const Checkbox = ({
           {...inputValues}
           indeterminate={isIndeterminate}
           onChange={onChange}
+          onClick={
+            onClick as unknown as React.MouseEventHandler<HTMLButtonElement>
+          }
           required={isRequired}
           inputProps={{
             "data-se": testId,
+            "aria-readonly": isReadOnly,
+            readOnly: isReadOnly,
           }}
```

Comment:
Can we memoize these `inputProps`? I can't see the whole file, so it might not make sense, but it looks like it could be memoized. If it's in `Field`'s function, then it's already memoized. My bad.

Reply:
We could memoize them, but since `aria-readonly` is being used on the native element via `inputProps`, and `isReadOnly` is already in memory, since it's used elsewhere, I wonder if there is much of a benefit of it. What do you think @KevinGhadyani-Okta?